src/components/PanelTable.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div class="panelTable">
+    <v-text-field
+      v-model="search"
+      append-icon="mdi-magnify"
+      label="Search"
+      single-line
+      hide-details
+    ></v-text-field>
+    <v-data-table
+      v-model="selected"
+      :headers="headers"
+      :items="lines"
+      item-key="index"
+      show-select
+      :items-per-page="-1"
+      :single-select="false"
+      class="elevation-1"
+      :search="search"
+    >
+      <template v-slot:item.element.sentence="props">
+        <v-edit-dialog :return-value.sync="props.item.element.sentence">
+          {{ props.item.element.sentence }}
+          <template v-slot:input>
+            <v-text-field
+              v-model="props.item.element.sentence"
+               :rules="[max25chars]"
+              label="Edit"
+              single-line
+              counter
+            ></v-text-field>
+          </template>
+        </v-edit-dialog>
+      </template>
+
+      <template v-slot:item.element.comboIndex="{ item }">
+        <v-btn @click="changeComboIndex(item.index, -1)" elevation="1" x-small
+          >&lt;</v-btn
+        >
+        {{ item.element.comboIndex }}
+        <v-btn @click="changeComboIndex(item.index, 1)" elevation="1" x-small
+          >&gt;</v-btn
+        >
+      </template>
+    </v-data-table>
+
+    <md-card-actions>
+      <md-button
+        class="md-raised md-primary"
+        @click="$store.dispatch('NEW_EMPTY_SENTENCE', lines.length)"
+        >Add</md-button
+      >
+      <md-button
+        v-if="$store.state.selected.length > 0"
```

Comment:
Plutôt directement lier le booléen à `disabled`
```suggestion
        :disabled="$store.state.selected.length === 0"
```